raiden-dapp/package.json

```diff
@@ -18,6 +18,7 @@
     "serve:no-logging": "export NODE_ENV=nologging && vue-cli-service serve"
   },
   "dependencies": {
+    "@babel/preset-typescript": "^7.12.1",
```

Comment:
These babel & cypress plugins should be a devDependency. They aren't needed for running the dApp, only developing/testing it.

Reply:
Didn't notice they were there. I'll fix it.